book/src/user/lightwalletd.md

```diff
@@ -0,0 +1,248 @@
+# Running lightwalletd with zebra
+
+Starting on [v1.0.0-beta.10](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.10), the Zebra RPC methods are fully featured to run a lightwalletd service backed by zebrad.
+
+Contents:
+
+- [Download and build Zebra](#download-and-build-zebra)
+- [Configure zebra for lightwalletd](#configure-zebra-for-lightwalletd)
+  - [RPC section](#rpc-section)
+  - [State section](#state-section)
+  - [Network section](#network-section)
+- [Sync Zebra](#sync-zebra)
+- [Download and build lightwalletd](#download-and-build-lightwalletd)
+- [Sync lightwalled](#sync-lightwalled)
+- [Run tests](#run-tests)
+- [Connect wallet to lightwalletd](#connect-wallet-to-lightwalletd)
+  - [Download and build the cli-wallet](#download-and-build-the-cli-wallet)
+  - [Run the wallet](#run-the-wallet)
+- [Lightwalletd and nginx](#lightwalletd-and-nginx)
+
+## Download and build Zebra
+[#download-and-build-zebra]: #download-and-build-zebra
+
+```console
+git clone https://github.com/ZcashFoundation/zebra
+cd zebra
+cargo build --release
+```
+
+Zebra binary will be at `target/release/zebrad`.
+
+## Configure zebra for lightwalletd
+[#configure-zebra-for-lightwalletd]: #configure-zebra-for-lightwalletd
+
+We need a zebra configuration file. First, we create a file with the default settings:
+
+```console
+./target/release/zebrad generate -o zebrad.toml
+```
+
+This will create a `zebrad.toml` file. Tweak the following options in order to prepare for lightwalletd setup.
+
+### RPC section
+[#rpc-section]: #rpc-section
+
+This change is required for zebra to behave as an RPC endpoint. The standard port for RPC endpoint is `8232`.
+
+```
+[rpc]
+listen_addr = "127.0.0.1:8232"
+```
+
+### State section
+[#state-section]: #state-section
+
+Make sure your cache is in a hard drive with enough space to hold the full blockchain (around 30G at the moment of writing - 2022-05-28).
+
+```
+[state]
+cache_dir = "/lot/of/free/space/for/zebra"
+...
+```
+
+### Network section
+[#network-section]: #network-section
+
+Only change this if you want to sync for the testnet:
+
+```
+[network]
+...
+network = 'Testnet'
+...
+```
```

Comment:
These steps are optional, can you please put them in an "advanced usage" section at the end?

Reply:
I removed this sections from the document in https://github.com/ZcashFoundation/zebra/pull/4526/commits/2bdaaa108bff4f5f6a3fa12765d3728d2fce9399